tools/tm-bench/main.go

```diff
@@ -93,6 +94,13 @@ Examples:
 		"broadcast_tx_"+broadcastTxMethod,
 	)
 
+	//catch Interrupt and quit tm-bench
+	go cmn.TrapSignal(func() {
```

Comment:
It doesn't seem advisable to further use this method, it logs uncontrolled and the functionality is simple enough that it could be copied as is.